packages/core/src/App/Containers/Redirect/__tests__/redirect.spec.jsx

```diff
@@ -0,0 +1,42 @@
+import React from 'react';
+import { createBrowserHistory } from 'history';
+import { Router } from 'react-router-dom';
+import { render } from '@testing-library/react';
+import Redirect from '../redirect';
+
+jest.mock('Stores/connect', () => ({
+    __esModule: true,
+    default: 'mockedDefaultExport',
+    connect: () => Component => Component,
+}));
+
+describe('<Redirect />', () => {
+    const history = createBrowserHistory();
```

Comment:
@aum-deriv could we recreate history in `beforeEach` that in every test case (when they will be added in the future) we will have new/fresh history

```
let history, mock_props;

    beforeEach(() => {
        history = createBrowserHistory()

        mock_props = {
            loginid: 'CR1234',
            verification_code: {
                payment_withdraw: '',
            },
            setVerificationCode: jest.fn(),
            setNewEmail: jest.fn(),
            setLoginId: jest.fn(id => {
                return id;
            }),
        };
    });
```